release/release_after_kokoro.sh

```diff
@@ -0,0 +1,195 @@
+
+set -x
+javap -private -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.appengine.login_0.1.0.*.jar \
+       -constants com.google.cloud.tools.eclipse.appengine.login.Constants
+javap -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.usagetracker_0.1.0.*.jar \
+       -constants com.google.cloud.tools.eclipse.usagetracker.Constants
+set +x
+
+echo "Check the output above. "
+ask_proceed
+
+###############################################################################
+echo
+echo "#"
+echo "# Run Eclipse from the command line to generate the following files:"
+echo "#"
+echo "#     $LOCAL_REPO/artifacts.xml"
+echo "#     $LOCAL_REPO/content.xml"
+echo "#     $LOCAL_REPO/features/*"
+echo "#     $LOCAL_REPO/plugins/*"
+echo "#"
+ask_proceed
+
+echo "The following steps require running an Eclipse binary."
+while true; do
+    echo -n "Enter the Eclipse home path: "
+    read ECLIPSE_HOME
+    if [ -x "$ECLIPSE_HOME/eclipse" ]; then break; fi
+    echo "'$ECLIPSE_HOME/eclipse' is not an executable. Try again."
+done
+
+set -x
+$ECLIPSE_HOME/eclipse -nosplash -consolelog \
+    -application org.eclipse.equinox.p2.publisher.FeaturesAndBundlesPublisher \
+    -metadataRepositoryName 'Google Cloud Tools for Eclipse' \
+    -metadataRepository file:$LOCAL_REPO \
+    -artifactRepositoryName 'Google Cloud Tools for Eclipse' \
+    -artifactRepository file:$LOCAL_REPO \
+    -source $SIGNED_DIR \
+    -publishArtifacts
+set +x
+
+if [ ! -e "$LOCAL_REPO/artifacts.xml" -o ! -e "$LOCAL_REPO/content.xml" ]; then
+    echo "The files have not been generated. Halting."
+    exit 1
+fi
+
+###############################################################################
+echo
+echo "#"
+echo "# Copy 'gs://gcloud-for-eclipse-testing/category.xml'"
+echo "# into '$WORK_DIR'."
+echo "#"
+ask_proceed
+
+set -x
+gsutil cp gs://gcloud-for-eclipse-testing/category.xml $WORK_DIR
+set +x
+
+if [ ! -e "$WORK_DIR/category.xml" ]; then
+    echo "The file was not copied. Halting."
+    exit 1
+fi
+
+###############################################################################
+echo
+echo "#"
+echo "# Run org.eclipse.equinox.p2.publisher.CategoryPublisher."
+echo "#"
+ask_proceed
+
+set -x
+$ECLIPSE_HOME/eclipse \
+    -nosplash -console -consolelog \
+    -application org.eclipse.equinox.p2.publisher.CategoryPublisher \
+    -metadataRepository file:$LOCAL_REPO \
+    -categoryDefinition file:$WORK_DIR/category.xml \
+    -categoryQualifier
+set +x
+
+###############################################################################
+echo
+echo "#"
+echo "# Upload the following newly created files"
+echo "#"
+echo "#     $LOCAL_REPO/artifacts.xml"
+echo "#     $LOCAL_REPO/content.xml"
+echo "#     $LOCAL_REPO/features/*"
+echo "#     $LOCAL_REPO/plugins/*"
+echo "#"
+echo "# into 'gs://cloud-tools-for-eclipse/<VERSION>/'."
+echo "#"
+ask_proceed
+
+while true; do
+    echo -n "Enter version: "
+    read VERSION
+    echo -n "Verify version: "
```

Comment:
Is it really necessary to verify the version? Or maybe only verify the version if it looks off via regexp?